app/models/acl_dao.rb

```diff
@@ -0,0 +1,91 @@
+class AclDao
+  attr_accessor :access_token, :client_id, :base_url
+
+  def initialize(token, client_id, base_url)
+    @access_token = token
+    @client_id = client_id
+    @base_url = base_url
+  end
+
+  def get_role(user_id)
+
+    # Sets whether the user is a specific role type
+    # Note: user could be all three.
+    user_is_daac_curator = false
+    user_is_arc_curator = true
```

Comment:
should this be `true`? does this mean they automatically get this role?

Reply:
Yes, that was my logic at the time, but I've been rethinking this today and going to talk with Alicia, but I think given anyone can get a earth data login account, we shouldn't default to arc curator like the existing application does.   In fact, if they have no ACLs, we should block access.   I'm going to confirm this but I think that is probably the way we should go.